jenkins/artifacts/JenkinWorkflow

```diff
@@ -1,112 +1,137 @@
 pipeline {
     agent {label 'buildserver'}
 
+    parameters {
```

Comment:
can we combine these 2 jenkins file? most lines are same?

Reply:
@cdurai-netapp was this commented addressed? do you think this makes sense?

Reply:
yes it make sense. working on it.

Reply:
fixed and sent it for review.